MCPB as the preferred method for installing MCP in Claude Desktop (#8445)
fishtown-analytics/docs.getdbt.com: website/snippets/_static-subdomain-required.md

:::info Static subdomains required

> Only accounts with static subdomains (for example, `abc123.us1.dbt.com`) can use OAuth with MCP servers. All accounts are in the process of being migrated to static subdomains by December 2025. Contact support for more information.

Reviewer: hey @DevonFulcher  do we mean december 2025 since the date has passed?

Author: Ooh good catch. I just copy-pasted this. Let me check if this is still relevant.

Reviewer 2: Fixed in a separate PR

> Only accounts with static subdomains (for example, `abc123.us1.dbt.com`) can use OAuth with MCP servers. All accounts are in the process of being migrated to static subdomains by December 2025. Contact support for more information.

Reviewer 2: ```suggestion
:::info Static subdomains required

Only accounts with static subdomains (for example, `abc123` in `abc123.us1.dbt.com`) can use OAuth with MCP servers. To find your account subdomain, navigate to **Account Settings** > **Account** > **Access URL**. If your account does not have a subdomain, contact support for more information.

:::
```

:::info Static subdomains required

> :::info Static subdomains required

Reviewer 2: ```suggestion
```

Only accounts with static subdomains (for example, `abc123` in `abc123.us1.dbt.com`) can use OAuth with MCP servers. To find your account subdomain, navigate to **Account Settings** > **Account** > **Access URL**. If your account does not have a subdomain, contact support for more information.

> :::

Reviewer 2: ```suggestion
```
